EventKit/eventkit-cloud
# Arcgis styles

## Description of Improvement

Fixes issue with cache getting service capabilities.
Adds logic to request feature services by specific layer. 
Adds a layer file for arcgis feature service sources in the data directory when using geopackage.  

## How to test

create a datapack using an arcgis feature service (mapService specifically to get the styles), and check the data folder for a lyrx file.  Those files can now be used directly into arcgis pro or can still be added using the create_aprx script. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
